src/shared/components/footer.jsx

```diff
@@ -43,10 +43,18 @@ const Footer = ({ showIcon = true }) => (
           .
         </p>
       )}
-      {// only display terms of service on an official mozilla installation
-      process.env.IODIDE_PUBLIC && (
+      {process.env.IODIDE_PUBLIC && (
+        // only display terms of service on an official mozilla installation
         <ul>
           <li>Alpha Software</li>
+          <li>
```

Comment:
We should incorporate the revision into the same li tag as "Alpha Software", adding a seperate tag adds unnecessary whitespace.